cli/src/test/java/com/devonfw/tools/ide/tool/jasypt/JasyptTest.java

```diff
@@ -0,0 +1,80 @@
+package com.devonfw.tools.ide.tool.jasypt;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+/**
+ * Integration test of {@link Jasypt}.
+ */
+public class JasyptTest extends AbstractIdeContextTest {
+
+  private static final String PROJECT_JASYPT = "jasypt";
+
+  @Test
+  public void testJasypt() {
+
+    // arrange
+    IdeTestContext context = newContext(PROJECT_JASYPT);
+    Jasypt commandlet = new Jasypt(context);
+
+    // act - install
+    commandlet.install();
+
+    // assert - install
+    checkInstallation(context);
+
+    // act and assert - run
+    runNoArgs(context, commandlet, null);
+    runRightArgs(context, commandlet, List.of("encrypt", "master", "secret"));
+    runRightArgs(context, commandlet, List.of("decrypt", "master", "secret"));
+    runWrongArgs(context, commandlet, List.of("wrong args"));
+  }
+
+  public void runNoArgs(IdeTestContext context, Jasypt commandlet, List<String> arguments) {
+
+    String expectedMessage = """
+        Jasypt encryption tool
+        Usage:
+         encrypt  <masterpassword>  <secret>             encrypt a secret with a master-password
+         decrypt  <masterpassword>  <secret>             decrypt an encrypted secret with a master-password
+        """;
+    commandlet.arguments.setValue(arguments);
+    commandlet.run();
+    assertLogMessage(context, IdeLogLevel.INFO, expectedMessage);
+  }
```

Comment:
Sorry, you have implemented all the tests and now things have to change slightly rendering that test work void...
BTW: It is much better to write multiple `@Test` methods for individual aspects instead of writing one big `testJasypt()` method that then tests various other things. Have a look at other existing tests and how they do it.